spec/iasm.dd

```diff
@@ -26,6 +26,38 @@ $(SPEC_S D x86 Inline Assembler,
         $(D D_InlineAsm_X86_64) version identifiers, respectively.
     )
 
+$(H2 $(LNAME2 asmstatements, Asm statement))
+
+$(GRAMMAR
+$(GNAME AsmStatement):
+    $(D asm) $(GLINK FunctionAttributes)$(OPT) $(D {) $(GLINK AsmInstructionList)$(OPT) $(D })
+
+    $(GNAME AsmInstructionList):
+        $(GLINK AsmInstruction) $(D ;)
+        $(GLINK AsmInstruction) $(D ;) $(I AsmInstructionList)
+    )
+
+    $(P Assembler instructions are located in an `asm` statement.
```

Comment:
I think this statement should be a little stronger, e.g., "Assembler instructions **must** be in an `asm` statement."